examples/push/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/config"
+)
+
+func main() {
+	if len(os.Args) != 4 {
+		fmt.Printf("Usage: %s\n <repo path> <remote url> <refspecs>", os.Args[0])
+		os.Exit(1)
+	}
+
+	repoPath := os.Args[1]
+	remoteName := "example-remote"
+	remoteUrl := os.Args[2]
+	refspec := config.RefSpec(os.Args[3])
+
+	if !refspec.IsValid() {
+		fmt.Printf("invalid refspec: %s\n", refspec)
+		os.Exit(1)
+	}
+
+	repo, err := git.NewFilesystemRepository(repoPath)
+	if err != nil {
+		panic(err)
+	}
+
+	if _, err := repo.CreateRemote(&config.RemoteConfig{
+		Name: remoteName,
+		URL:  remoteUrl,
+	}); err != nil {
+		panic(err)
+	}
+
+	if err := repo.Push(&git.PushOptions{
```

Comment:
And is supposed to illustrate the code with his homologue at git 
`Info("git clone %s %s", url, directory)`